### PR TITLE
vmm:support running stratovirt sandbox in x86

### PR DESCRIPTION
**Pull request by Vanient**
Before we just support running sandbox using stratovirt as hypervisor in arm, now also support in x86.

### CODE REVIEW

File: vmm/sandbox/src/stratovirt/config.rs [hunk 1]
```diff
@@ -330,7 +355,10 @@ mod tests {
             "-kernel",
             "/var/lib/kuasar/vmlinux.bin",
             "-append",
+            #[cfg(target_arch = "aarch64")]
             "console=hvc0 console=hvc1 iommu=off debug panic=1 pcie_ports=native root=/dev/vda1 ro rootfstype=ext4",
+            #[cfg(target_arch = "x86_64")]
+            "console=hvc0 console=hvc1 iommu=off debug panic=1 pcie_ports=native root=/dev/vda ro rootfstype=ext4",
```
Comment (Burning1020) [inline vmm/sandbox/src/stratovirt/config.rs:361]: Is it possiable to reuse `ROOTFS_KERNEL_PARAMS` insteadof `cfg` again?

File: vmm/sandbox/src/vm.rs [hunk 2]
```diff
@@ -125,6 +127,7 @@ impl Default for HypervisorCommonConfig {
             image_path: "".to_string(),
             initrd_path: "".to_string(),
             kernel_params: "".to_string(),
+            firmware: "".to_string(),
             enable_mem_prealloc: false,
```
Comment (Burning1020) [inline vmm/sandbox/src/vm.rs:131]: Suggestion, not necessary:
```rust
Self {
    debug: false,
    vcpus: 1,
    memory_in_mb: 1024,
    kernel_path: "/var/lib/kuasar/vmlinux.bin".to_string(),
    ..Default::default()
}
```

**Comment by Burning1020** [comment]
Still need some descriptions in pr

File: vmm/sandbox/src/stratovirt/config.rs (hunk 1 above)
Comment (Vanient) [reply vmm/sandbox/src/stratovirt/config.rs]: fixed

File: vmm/sandbox/src/vm.rs (hunk 2 above)
Comment (Vanient) [reply vmm/sandbox/src/vm.rs:131]: I tried but it caused default recursive call error

**Review by flyflypeng — APPROVED** [review]

**Pull request merged**